src/host_abi.zig

```diff
@@ -65,7 +65,7 @@ pub fn RocOps(comptime CallEnv: type, comptime HostFns: type) type {
 /// If it cannot provide a non-null pointer (e.g. due to OOM), it
 /// must not return, and must instead do something along the lines
 /// of roc_crashed.
-pub const RocAlloc = struct {
+pub const RocAlloc = extern struct {
     alignment: usize,
```

Comment:
Not sure if this helps here since the struct has no padding, but in Zig std lib we've started using an [Alignment](https://ziglang.org/documentation/0.14.0/std/#std.mem.Alignment) type that stores the log2 value. It's pretty handy because you only need 6 bits to describe any alignment for a 64-bit address.

Reply:
Nice, that's a cool trick! I agree that it probably doesn't help here, but that's a good one to know for future reference. 🤘